health/REFERENCE.md

```diff
@@ -895,6 +895,48 @@ lookup: mean -10s of user
 
 Since [`z = (x - mean) / stddev`](https://en.wikipedia.org/wiki/Standard_score) we create two input alarms, one for `mean` and one for `stddev` and then use them both as inputs in our final `cpu_user_zscore` alarm.
 
+### Example 8 - Anomaly rate based CPU dimensions alarm
+
+Warning if 5 minute rolling anomaly rate for any CPU dimension is above 5%, critical if it goes above 20%:
```

Comment:
Can we make anomaly rate a hyperlink that redirects to a doc that describes what "anomaly rate" is? As it is now it implies a person has knowledge of the subject.

Reply:
done